Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -917,6 +921,20 @@ bool HTMLImageElement::isSystemPreviewImage() const
 }
 #endif
 
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+bool HTMLImageElement::isMultiRepresentationHEIC() const
+{
+    if (!m_sourceElement)
+        return false;
+
+    auto& typeAttribute = m_sourceElement->attributeWithoutSynchronization(typeAttr);
+    if (typeAttribute != MULTI_REPRESENTATION_HEIC_MIME_TYPE_STRING)
```

Comment:
can this just be return typeAttribute == MULTI_REPRESENTATION_HEIC_MIME_TYPE_STRING ?

Reply:
Yup.